app/controllers/api/v1/allocations_controller.rb

```diff
@@ -15,8 +15,14 @@ def index
 
       def create
         allocation = Allocation.new(allocation_attributes)
+        allocation.moves = moves if allocation.valid?
+
         allocation.save!
 
+        allocation.moves.each do |move|
```

Comment:
amend to not send notifications unless allocation/moves are saved and valid

Reply:
I was thinking that might be worth to move these calls to `Notifier` to a `after_action`, since are not the main goal of the actions.